[2022-12-06 14:28:31,607] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:27:16.856178+00:00 [queued]>
[2022-12-06 14:28:31,640] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:27:16.856178+00:00 [queued]>
[2022-12-06 14:28:31,640] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:28:31,640] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-12-06 14:28:31,641] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:28:31,654] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): LoadPositionsInMongoDB> on 2022-12-06T14:27:16.856178+00:00
[2022-12-06 14:28:31,661] {standard_task_runner.py:52} INFO - Started process 199 to run task
[2022-12-06 14:28:31,669] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_mongo_db_dag', 'LoadPositionsInMongoDB', '2022-12-06T14:27:16.856178+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_mongo_db_dag.py', '--cfg-path', '/tmp/tmp26k1760c', '--error-file', '/tmp/tmpvsiut6r9']
[2022-12-06 14:28:31,671] {standard_task_runner.py:77} INFO - Job 7: Subtask LoadPositionsInMongoDB
[2022-12-06 14:28:31,750] {logging_mixin.py:104} INFO - Running <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:27:16.856178+00:00 [running]> on host db4794db4ef9
[2022-12-06 14:28:32,027] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_mongo_db_dag
AIRFLOW_CTX_TASK_ID=LoadPositionsInMongoDB
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T14:27:16.856178+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-06T14:27:16.856178+00:00
[2022-12-06 14:28:32,275] {logging_mixin.py:104} INFO - debut de chargement des données dans mongodb
[2022-12-06 14:28:32,681] {logging_mixin.py:104} INFO - fin de chargement des données dans mongodb
[2022-12-06 14:28:32,687] {python.py:151} INFO - Done. Returned value was: None
[2022-12-06 14:28:32,704] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_mongo_db_dag, task_id=LoadPositionsInMongoDB, execution_date=20221206T142716, start_date=20221206T142831, end_date=20221206T142832
[2022-12-06 14:28:32,736] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-06 14:28:32,773] {local_task_job.py:151} INFO - Task exited with return code 0
